assets/src/apps/delivery/Delivery.tsx

```diff
@@ -107,8 +110,26 @@ const Delivery: React.FC<DeliveryProps> = ({
     return () => clearTimeout(timer);
   }, [screenIdleExpirationTime]);
 
+  const handleUserThemePreferende = () => {
```

Comment:
changes on this page is related to [Add "dark" mode to adaptive pages](https://eliterate.atlassian.net/browse/MER-3523). I used the same code and logic that is used across Torus.